### PR TITLE
Add context menu entry to remove all items from a bookmark folder

### PR DESCRIPTION
**Pull request by aekeus**
Implements #1774

### CODE REVIEW

File: js/state/siteUtil.js [hunk 1]
```diff
@@ -136,6 +136,23 @@ module.exports.removeSite = function (sites, siteDetail, tag) {
...
+ * @param sites The application state's Immutable sites list
+ * @param siteDetail The siteDetail to remove
+ * @return The new sites Immutable object
+ */
+module.exports.emptySite = function (sites, siteDetail, tag) {
+  const index = module.exports.getSiteIndex(sites, siteDetail, tag)
+  if (index === -1) {
+    return sites
+  }
+  return sites.filter((site) => {
```
Comment (bbondy) [inline js/state/siteUtil.js]: We should leave the site item here and just remove the bookmark tag from it.  That way it still stays in history.

**Comment by bbondy** [comment]
Looks great otherwise, just one small change. Thanks!

File: js/state/siteUtil.js (hunk 1 above)
Comment (aekeus) [inline js/state/siteUtil.js]: Ok, right. I didn't think about that. Will do.

**Comment by bbondy** [comment]
this needs rebasing too when fixed

**Comment by bbondy** [comment]
since this is more than a month old I'll just close for now. Please re-issue a pull request if/when you get back to it. Thanks!

**Comment by aekeus** [comment]
Thanks will do.

**Pull request closed without merging**